cases/operator-crashloop-missing-perms/deployment.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: rbac-test
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: rbac-test
+  template:
+    metadata:
+      labels:
+        app: rbac-test
+    spec:
+      serviceAccountName: restricted-sa
+      containers:
+      - name: kubectl-runner
```

Comment:
this is not what this is supposed to test.  What you need to do is find a standard kubernetes operator and break its rbac settings.  They normally crash entirely